pkgs/by-name/st/stackql/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  testers,
+  stackql,
+}:
+
+buildGoModule rec {
+  pname = "stackql";
+  version = "0.5.643";
+
+  versionComponents = lib.splitVersion version;
```

Comment:
these don't need to be derivation attributes, i would either inline them or use `let` block

Reply:
removed